src/server/pps/cmds/cmds.go

```diff
@@ -539,6 +539,42 @@ each datum.`,
 	shell.RegisterCompletionFunc(listDatum, shell.JobCompletion)
 	commands = append(commands, cmdutil.CreateAliases(listDatum, "list datum", datums))
 
+	var since string
+	kubeEvents := &cobra.Command{
+		Use:   "{{alias}}",
+		Short: "Return the kubernetes events.",
+		Long:  "Return the kubernetes events.",
+		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			client, err := pachdclient.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			since, err := time.ParseDuration(since)
+			if err != nil {
+				return errors.Wrapf(err, "error parsing since(%q)", since)
```

Comment:
Errors need not contain the word "error".  `parse since(%q)` is fine.

Reply:
will change this